Makefile

```diff
@@ -28,9 +28,13 @@ TESTARGS_DEFAULT := "-v"
 export TESTARGS ?= $(TESTARGS_DEFAULT)
 VERSION ?= $(shell cat COMPONENT_VERSION 2> /dev/null)
 IMAGE_NAME_AND_VERSION ?= $(REGISTRY)/$(IMG)
+CONTROLLER_NAME = $(shell cat COMPONENT_NAME 2> /dev/null)
+CONTROLLER_NAMESPACE ?= open-cluster-management
 # Handle KinD configuration
 CLUSTER_NAME ?= hub
-KIND_NAMESPACE ?= open-cluster-management
+KIND_NAME ?= test-$(CLUSTER_NAME)
+KIND_CLUSTER_NAME ?= kind-$(KIND_NAME)
```

Comment:
I think these are used/defined in the common Makefile and may not be used directly here?

```suggestion
```